### ISSUE TITLE
Not working in dev setup

### ISSUE THREAD

**Issue by phase7** [mohidex/openweather-api#19]
I've followed the local development setup, it shows the following error
![image](https://github.com/mohidex/openweather-api/assets/19484371/24c855e0-5008-4718-afc6-43e8594a2f80)

**Comment by mohidex** [comment]
Yes! Thats right. The application has been changed. You have to run the loader to load all the weather reports and then you can find the weather.

The current dev setup will work before `76faa087f06ab02f595306f8eb4e5ea9e0ae786a` commit tag.
